src/gpuarray_buffer_opencl.c

```diff
@@ -90,9 +90,10 @@ cl_ctx *cl_make_ctx(cl_context ctx) {
   res->refcnt = 1;
   res->exts = NULL;
   res->blas_handle = NULL;
-  res->q = clCreateCommandQueue(ctx, id,
-				qprop&CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE,
-				&err);
+  res->q = clCreateCommandQueue(
```

Comment:
ok. I don't want to spend time on this, but I find that strange. If this is something that should be added, make an issue for this. Otherwise, I'll be surprised again when I see this next time:)